app/controllers/assessments_controller.rb

```diff
@@ -63,15 +63,16 @@ def show
     perform_assessment
   rescue StandardError => err
     Sentry.capture_exception(err)
-    render json: Decorators::ErrorDecorator.new(err).as_json, status: :unprocessable_entity
+    render json: Decorators::V3::ErrorDecorator.new(err).as_json, status: :unprocessable_entity
```

Comment:
Will check as review continues... do we need a 
```ruby
Decorators::V4::ErrorDecorator
```
too?

Reply:
I don't think we do - the error format is identical, so we can just use that one. I suppose it might make sense to move it out of the V3 namespace and just call it `Decorators::ErrorDecorator` again.